WeatherSystem.Test/UserShould.cs

```diff
@@ -0,0 +1,83 @@
+﻿using WeatherSystem.Strategies;
+
+namespace WeatherSystem.Test
+{
+
+    public class UserShould
+    {
+        [Theory]
+        [InlineData("Nablus" , 32 , 40.1)]
+        public void Parse_JSON_Reurtn_Same_Location(string actualLocation , float actualTempreture , float actualHumidity)
```

Comment:
u can use MemberData instead of InlineData to pass all data u need from the theory method u will create
in this case u can pass `weatherData` and `context ` from it and just do Act and Assert in the test